packages/extra/test/typescript_definitions/js-joda-extra-tests.ts

```diff
@@ -88,8 +88,10 @@ function test_DayOfYear() {
 function test_Interval() {
     const instant = Instant.now();
     const duration = Duration.ofHours(1);
+    const endExclusiveOrDuration: Instant | Duration = duration;
 
     Interval.of(instant, duration);
+    Interval.of(instant, endExclusiveOrDuration);
```

Comment:
What about testing `Interval.of(instant, endExclusiveInstant);` as a third case ?

Reply:
that's already in here, it's just not showing in the diff. https://github.com/js-joda/js-joda/pull/652/files#diff-1c99bd2ff528e18c76d17c2f5f739141de1f996ddd78696f35ac965c3390a1d4R98

Reply:
You are testing two times with the same instance `duration`, what i miss here is a test with a value of class Instant, eg  `Interval.of(instant, instant)`

Reply:
yes, because it's on a different line, it was already there. For some reason it's way down. Do you want me to move it up? Trying to make as few changes as possible.

Reply:
Ah ok, there it is, yes maybe we should move `Instant.parse` before the `Instant.of` tests